pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -828,7 +828,10 @@ default-package-overrides:
   - ghost-buster ==0.1.1.0
   - gi-atk ==2.0.18
   - gi-cairo ==1.0.17
+  - gi-dbusmenu ==0.4.4
```

Comment:
You must insert those constraints before the `# LTS Haskell 13.26` comment, otherwise you changes will be lost the next time https://github.com/NixOS/cabal2nix/blob/master/update-stackage.sh runs to update the Stackage version set.